src/styles/components/pods-view.less

```diff
@@ -0,0 +1,125 @@
+.pod-instances-table {
+
+  tbody {
+    line-height: 1.8;
+  }
+
+}
+
+.pod-instances-container-id {
+  color: @white;
+  margin: @base-spacing-unit * -1/4 0;
+  padding: @base-spacing-unit * 1/4 @base-spacing-unit * 1/2;
+
+  &.is-expandable {
```

Comment:
This is done! I created an `expanding-table` stylesheet as I probably should have done to begin with.